src/common/containers/__tests__/AggregationFiltersContainer.test.jsx

```diff
@@ -58,6 +59,35 @@ describe('AggregationFiltersContainer', () => {
             },
           },
         },
+        total: 1,
+      }),
+    });
+    const wrapper = shallow(
+      <AggregationFiltersContainer store={store} />
+    ).dive();
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('does not render aggregations when numberOfResults is 0', () => {
+    const store = getStoreWithState({
+      search: fromJS({
+        aggregations: {
+          agg: {
+            buckets: [{}],
+            meta: {
+              title: 'Jessica Jones',
+              order: 1,
+            },
+          },
+          emptyAgg: {
+            buckets: [],
+            meta: {
+              title: 'Luke Cage',
+              order: 2,
+            },
+          },
+        },
+        total: 0,
```

Comment:
this test cases is a bit hairy.. the aggregations here will not be displayed because are empty. It's better to add aggregations which are suppose to be render and make sure they don't because we have ``total: 0``